File: Array/Searching/LinearSearch.py
def linear_search(arr, target):
    for element in arr:
        if element == target:
            print("Element found in an Array")
    print("Element not found")

# Example usage
if __name__ == "__main__":
    arr = [1, 2, 3, 4, 5]
    target = 3
    result = linear_search(arr, target)
    print(f"Element found at index: {result}")

File: Array/Searching/LinearSearch.go
package main

import "fmt"

func linearSearch(arr []int, target int) {
    for _, value := range(arr) {
        if value == target {
            fmt.Println("Element found in an array")
			return
        }
	}
    fmt.Println("Element not found")

}

func linearSearchMain() {
    arr := []int{1, 2, 3, 4, 5}
    target := 3
    linearSearch(arr, target)
}